src/jit/jiteh.cpp

```diff
@@ -637,6 +660,59 @@ bool            Compiler::ehIsBlockEHLast(BasicBlock* block)
            (ehIsBlockHndLast(block) != nullptr);
 }
 
+/******************************************************************************************
+* Return the EH descriptor for the most nested region that may handle exceptions raised in this BasicBlock
+* (or nullptr if this block's exceptions propagate to caller).
+*/
+EHblkDsc*       Compiler::ehGetBlockExnFlowDsc(BasicBlock* block)
+{
+  EHblkDsc* hndDesc = ehGetBlockHndDsc(block);
+
+  if ((hndDesc != nullptr) && hndDesc->InFilterRegionBBRange(block))
+  {
+    // If an exception escapes a filter, or if 'false' is returned at
+    // the end of a filter, the (original) exception is propagated to
+    // the next outer handler.  The "next outer handler" is the handler
```

Comment:
How about "If an exception is thrown in a filter (or escapes a callee in a filter), or if exception_continue_search (0/false) is returned"?  Because
 1. While you can't have protected regions in a filter, you can have callees which can catch internal exceptions (or not).
 2. CLR filters can only use two of the return values, so IMO it's natural to think of the return as a bool (and in fact the C# construct expresses it as a bool)